extensions/registry.py

```diff
@@ -3410,6 +3410,13 @@
         'supporters' : { 'NVIDIA' },
         'url' : 'extensions/NV/NV_light_max_exponent.txt',
     },
+    'GL_NV_memory_attachment' : {
+        'number' : 524,
+        'esnumber' : 304,
```

Comment:
Also needs to change to 305 to match the extension spec document. Otherwise this LGTM.